templates/mip_install_rd_dna_config_-1.0-.yaml

```
@@ -43,20 +43,24 @@ emip:
     vt:
       version: gitRepo
   singularity:
-    cnvnator: 
+    cnvnator:
       executable:
         cnvnator:
         cnvnator2VCF.pl:
       uri: shub://Clinical-Genomics/MIP:cnvnator-0.4.1
-    tiddit:
-      executable:
-        TIDDIT.py:
-      uri: shub://J35P312/TIDDIT:2.8.0
     manta:
       executable:
         configManta.py:
         runWorkflow.py: "no_executable_in_image"
       uri: shub://Clinical-Genomics/MIP:manta-1.6.0
+    peddy:
+      executable:
+        peddy: "python -m peddy "
```

```suggestion
        peddy: "python -m peddy"
```
Or was the space needed?

---

Do not think so